apps/prairielearn/src/pages/instructorFileTransfer/instructorFileTransfer.ts

```diff
@@ -33,17 +37,14 @@ async function getFileTransfer(file_transfer_id, user_id) {
       } and ${user_id} (types: ${typeof file_transfer.user_id}, ${typeof user_id})`,
     );
   }
-  const courseResult = await sqldb.queryOneRowAsync(sql.select_course_from_course_id, {
-    course_id: file_transfer.from_course_id,
-  });
-  file_transfer.from_course = courseResult.rows[0];
-  return file_transfer;
+  const from_course = await selectCourseById(file_transfer.from_course_id);
+  return { ...file_transfer, from_course };
```

Comment:
Nit: consider moving the course selection into the route handler so that this can return a pure `FileTransfer`.

Reply:
Done in f7f56db0079eea6216b0c0375daca4ce5a79e78f.